site/cypress/e2e/Cypress-Gradeable/hotkey.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('TA grading hotkey testing', () => {
+    after(() => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'update?nav_tab=2']);
+        cy.get('[data-testid="peer-component-container"]').should('exist');
+        cy.get('[data-testid="peer-component-container"]').find('[data-testid="delete-gradable-component"]').click();
+        cy.logout();
+    });
+
+    it('toggle keyboard shortcut', () => {
+        cy.login();
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'details']);
+        cy.get('[data-testid="view-sections"]').click();
+        cy.get('[data-testid="grade-button"]').eq(12).click();
+        cy.get('[data-testid="grading-panel-header"]').as('navigationBar');
+        cy.get('@navigationBar').type('{A}');
```

Comment:
For the testing of the hotkeys, on my system it is flaky if you use the nav-bar as the place you are wanting to type. After some research, many people just do cy.get('body').type({letter}); to use hotkeys, and after testing with my system, it works more consistently. Can you test this on your machine, and maybe a couple of other people to see if it is just my machine ?

Reply:
yeah its working fine